### PR TITLE
Removing references to wellington region for compute service.

### PR DESCRIPTION
**Pull request by danielobyrne**

### CODE REVIEW

**Comment by adrianjarvis** [comment]
Check the API URLS in the examples too, e.g. "https://api.cloud.catalyst.net.nz:5000/" is the WLG endpont for Keystone.

File: source/backups/_scripts/duplicity-vars.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ export DUPLICITY_NUM_RETRIES='3'
 
 # Catalyst Cloud object storage credential information
 export SWIFT_USERNAME='<your-backup-user>@<your-project-name>'
-export SWIFT_REGIONNAME='nz_wlg_2'
+export SWIFT_REGIONNAME='nz-por-1'
 export SWIFT_TENANTNAME='<your-project-name>'
 export SWIFT_PASSWORD='<your-openrc-password>'
 export SWIFT_AUTHURL='https://api.cloud.catalyst.net.nz:5000'
```
Comment (adrianjarvis) [inline source/backups/_scripts/duplicity-vars.sh:16]: Change to "https://api.nz-por-1.catalystcloud.io:5000/"

File: source/best-practices/security-bp.rst [hunk 2]
```diff
@@ -65,7 +65,7 @@ censor the password before sending it, as done below.
 
   OS_AUTH_URL=https://api.cloud.catalyst.net.nz:5000/v3
```
Comment (adrianjarvis) [inline source/best-practices/security-bp.rst:66]: Change to "https://api.nz-por-1.catalystcloud.io:5000/"

File: source/object-storage/tutorial-scripts/replication-api.rst [hunk 3]
```diff
@@ -37,7 +37,7 @@ environment variable as "*OS_STORAGE_URL*" like so:
     $ export OS_STORAGE_URL="https://object-storage.nz-por-1.catalystcloud.io:443/v1/AUTH_1xxxxxxxxxxxxxxxxxxxxxxxxxxxxe54"
 
 After we have set our storage URL, we need to find the name of the policy we
-want our container to have. For this example we will use the wellington single
+want our container to have. For this example we will use the Wellington single
```
Comment (adrianjarvis) [inline source/object-storage/tutorial-scripts/replication-api.rst:40]: Can you please change this to use Porirua or Hamilton for the single region policy. You will also need to change the StorageURL in the output from "swift stat -v" below

File: source/best-practices/security-bp.rst (hunk 2 above)
Comment (danielobyrne) [reply source/best-practices/security-bp.rst]: updated this example and others to remove the v3 and to have the catalystcloud.io change

**Pull request merged**